### PR TITLE
[gtk3.20] Notebook tab minimal design update and tab border fix

### PR DESCRIPTION
**Pull request by megax**
image
image

Added hover->border.
Added other tab border.
Background color change.

### CODE REVIEW

**Comment by megax** [comment]
@lots0logs Minimal design change.

**Comment by megax** [comment]
@lots0logs Fixed other bug.
Old:
image

New:
image

**Comment by satya164** [comment]
Can you fix the code style as per the bot's comments?

**Comment by lots0logs** [comment]
Probably should ignore [this error](https://github.com/numixproject/numix-gtk-theme/pull/345#discussion_r60210470) as we aren't following it anywhere in the theme's code right now. I'm not very confident that GTK even supports the "proper" syntax of double colons for pseudo selectors. @satya164 can we disable that particular warning?

**Comment by satya164** [comment]
@lots0logs Yes, we should fix that one in https://github.com/numixproject/numix-gtk-theme/blob/master/.scss-lint.yml

**Comment by megax** [comment]
Code style fixed :)

**Comment by SammysHP** [comment]
@megax https://github.com/numixproject/numix-gtk-theme/pull/345#issuecomment-211860474

> Fixed other bug.

I prefer the old version with white text on orange background. Gray text is difficult to read.

**Comment by megax** [comment]
@SammysHP Not problem :) Backdrop mode screenshot... :P Color style normal (white text). Only toolbar background color fix.

**Comment by megax** [comment]
Example (Normal mode window, not backdrop):
image

**Comment by megax** [comment]
@lots0logs How is the revevision of the changes going? should i change something else? There is an unsolved problem wherre the border is missing, i have already mentionet it but in connection with nemo, theere are some place where it is missing. I have no clue about it( it is good in adwaita).

**Comment by lots0logs** [comment]
@megax It's not clear to me what problems these changes address. I havent seen any issues with the tabs since the most recent changes I made. Could you grab some screenshots of the issues you are seeing?

**Comment by megax** [comment]
Not complete the border.
Example:
image

image

**Comment by lots0logs** [comment]
We don't want to look like Adwaita though. The border looks tacky to me. Sorry :grimacing:

**Comment by megax** [comment]
I don't want to look Adwaita, only sample :)

**Comment by khurshid-alam** [comment]
@megax That border is probably nemo specific issue. Anywhere else where it has the same issue? Beside it looks better without the border. 

Stack Rhythmbox and other fixes look good. We can merge those.

**Comment by megax** [comment]
@khurshid-alam Not only nemo bug.

Example:
image

seahorse
image

transmission
image

gtk3-widget-factory :)
image

**Comment by SammysHP** [comment]
The problem is that only `tab` is styled but not the outer `tabs` or `header`.

```
...
├── notebook
│   ├── header
│   │   └── tabs
│   │       ├── tab
│   │       ├── tab
│   │       └── tab
│   └── stack
│       ├── box
│       ├── box
│       └── box
...
```

**Comment by megax** [comment]
@khurshid-alam, @SammysHP, @lots0logs Fixed bug... :) (border minimal color change)
image

File: gtk-3.20/scss/widgets/_notebook.scss [hunk 1]
```diff
@@ -30,13 +30,52 @@
         > header {
             padding: 0;
             margin: 0;
-            border-width: 0;
+            border-width: 1px;
             border-radius: 0;
+            border-color: shade($base_color, .8);
```
Comment (SammysHP) [inline gtk-3.20/scss/widgets/_notebook.scss]: AFAIK there is a variable for the border color.

`$borders_color`: https://github.com/numixproject/numix-gtk-theme/blob/master/gtk-3.20/scss/_global.scss#L16

Comment (megax) [inline gtk-3.20/scss/widgets/_notebook.scss]: $borders_color dark...

Comment (megax) [inline gtk-3.20/scss/widgets/_notebook.scss]: Fixed :)

**Comment by lots0logs** [comment]
I still don't see any issue with the tabs. @khurshid-alam is it just me? Do you see what they are saying?

**Comment by SammysHP** [comment]
The border is only as long as the tabs and not as the whole `tabs`/`header` area.

tabs

**Comment by khurshid-alam** [comment]
They want to decorate other tabs/header area inside notebook which all the other themes do.

**Comment by khurshid-alam** [comment]
@megax Rebase your code on top of current master. Did you fix the issue with incomplete border? It doesn't reflect with your changes.

**Comment by megax** [comment]
@khurshid-alam 

**Changes:**
- Tabs new background color
- Tabs border fix
- Remove 1px bg_color style border
- bottom, top, right, left tab selected_bg_color + 1px and transition
- Hover adding bottom, top, right, left tab border color
- Fixed arrow margin

**Before:**
anim
anim

**After:**
tab_all
anim

**Pull request merged**